test/extended/testdata/router/router-http2.yaml

```diff
@@ -45,168 +29,35 @@ objects:
       name: http2
   spec:
     containers:
-    - image: golang:1.14
+    - image: ${IMAGE}
       name: server
-      command: ["/workdir/http2-server"]
+      command: ["ingress-operator", "serve-http2-test-server"]
```

Comment:
There is another subcommand `serve-grpc-test-server` in router-grpc-interop.yaml

Reply:
Working as defined :)